src/linear_samplers/vec_col_one_rand_cyclic.jl

```diff
@@ -0,0 +1,50 @@
+# This file is pat of RLinearAlgebra.jl
+# 1. Specifies type
+# 2. Implements sample function
+# 3. Exports Type
+
+# using Random
+
+"""
+    LinSysVecColOneRandCyclic <: LinSysVecColSelect
+
+A mutable structure with a field to store a cycling order. Randomly specifies a cycling
+order over the equations of a linear system. Once this ordering is specified, the ordering
+is kept fixed.
+
+# Fields
+- `order::Union{Vector{Int64},Nothing}`
```

Comment:
I tried to simplify the type to just `Vector{Int64}`, which you can see in the `LinSysVecRowResidCyclic` method. The default is initialize it to `Int64[]`. This gets populated by the sample method

Reply:
@dmaldona you can see this [here](https://github.com/numlinalg/RLinearAlgebra.jl/blob/ebd03232a4af114e8c78b6204556a1d5b2860c78/src/linear_samplers/vec_row_cyclic_residual.jl#L15) for the `LinSysVecRowResidCyclic` method.

You can see how it is initialized then on this [line](https://github.com/numlinalg/RLinearAlgebra.jl/blob/ebd03232a4af114e8c78b6204556a1d5b2860c78/src/linear_samplers/vec_row_cyclic_residual.jl#L36) .

Reply:
@vp314 noted and modified